node/node/Cargo.toml

```diff
@@ -108,18 +106,3 @@ authority-selection-inherents = { workspace = true, features = ["mock"] }
 
 [features]
 default = ["sidechain-domain/std"]
-rocksdb = [
-	"sc-cli/rocksdb",
-	"sc-service/rocksdb",
-	"sidechain-runtime/with-rocksdb-weights",
-]
-paritydb = [
-	"sidechain-runtime/with-paritydb-weights",
-]
```

Comment:
I've removed the features to go leaner in our demo node. We use only rocksdb, so removing these features.